pkg/release/release.go

```diff
@@ -379,10 +402,12 @@ func (r *Release) install(client helm.Client, hr *v1.HelmRelease, chart chart, v
 		Wait:       hr.Spec.Wait,
 	})
 	if err != nil {
+		status.SetInstallPhase(r.hrClient.HelmReleases(hr.Namespace), hr, v1.HelmReleasePhaseFailed)
 		status.SetStatusPhase(r.hrClient.HelmReleases(hr.Namespace), hr, v1.HelmReleasePhaseFailed)
```

Comment:
Obsolete / duplicate of the above?

Reply:
This one is right, there are 3 phases:

Installing: Clear count to 0
Failed: Increment
Success: Clear count to 0

I could just re-use the `HelmReleasePhaseSucceeded` however in this case if desired.